packages/apps/halo-app/local-vault.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HALO</title>
+    <style>
+      html, body {
+        background: transparent !important;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/node_modules/@dxos/halo-app/src/vault/main.ts"></script>
```

Comment:
thats interesting, is this stable across dev and prod mode?

another idea is:
```suggestion
    <script type="module">
       import './src/main.ts' // relative to this html file
    </script>
```

Reply:
1. It doesn't need to be stable for prod because the expectation is that this would only be used for development, however that being said I suspect that it would work fine because during bundle vite rewrites this path based on the output of the bundle.
2. The path is not relative to the html file it's relative to where the vite server it being run which is why starting with node_modules is stable because this is only used by other packages where halo-app is a dev dep.